sdks/python/apache_beam/testing/test_stream.py

```diff
@@ -171,13 +184,20 @@ class TestStream(PTransform):
   time. After all of the specified elements are emitted, ceases to produce
   output.
   """
-  def __init__(self, coder=coders.FastPrimitivesCoder(), events=None):
+  def __init__(
```

Comment:
Please add pydoc comments mentioning the few important pieces:
1) specifying the output_tags allows for adding outputs that produce no events
2) output_tags must be a superset of tags found in events if events is specified